File: abc155/a/typescript/a.ts
import * as fs from "fs";

function main(){
  const [a, b, c] = fs.readFileSync("/dev/stdin", "utf-8").trim().split(/\s+/).map(Number);

  if(new Set([a, b, c]).size === 2){
    console.log("Yes");
  }
  else{
    console.log("No");
  }
}

main();

File: abc155/a/a.go
package main

import (
	"fmt"
)

func main() {
	var a, b, c int
	fmt.Scan(&a, &b, &c)

	if ((a == b) && (b == c)) || ((a != b) && (b != c) && (a != c)) {
		fmt.Println("No")
	} else {
		fmt.Println("Yes")
	}

	return
}
